main.go

```diff
@@ -6,32 +6,32 @@ import (
 	"gopkg.in/urfave/cli.v2"
 )
 
-var (
-	version string
-
-	InputFileFlag = &cli.StringFlag{
-		Name:    "input-file",
-		Aliases: []string{"in"},
-		Usage:   "file to convert from",
-	}
-
-	OutputFileFlag = &cli.StringFlag{
-		Name:    "output-file",
-		Aliases: []string{"out"},
-		Usage:   "file to write converted data to",
-	}
-
-	DelimiterFlag = &cli.StringFlag{
-		Name:    "delimiter",
-		Aliases: []string{"d"},
-		Value:   ",",
-		Usage:   "",
-	}
-)
-
 type row map[string]interface{}
 
 func main() {
+	var (
+		version string
+
+		InputFileFlag = &cli.StringFlag{
```

Comment:
The need to be made available outside of `func main()` in order to be used in `csvToJSON`. Though, I agree the scope should be limited which should be accomplished via "unexporting" them like `inputFileFlag = ...`

Reply:
the context gives you access to it, am i wrong?

Reply:
It does, but you get this by doing `context.String(... flag name ...)`. This goes back to the comment for line 16 for **csv_to_json.go**

Reply:
setting the string in const lets you rename gracefully in one place as well (if you want to rename for the user, you can use alias to handle retrocompatibility), and it has the benefit of getting rid of the global vars.
There might be something I don't see here?